metricbeat/module/logstash/_meta/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.elastic.co/logstash/logstash:6.0.0
+FROM docker.elastic.co/logstash/logstash:6.2.3
```

Comment:
We have to figure out a way to proactively tests on the snapshots release for that.

Reply:
Yeah, it is a general thing we have to come up with a solution, meaning testing each module against multiple versions.